docs/installation_configuration_guide/DeploymentManagerConfiguration.md

```diff
@@ -41,10 +39,11 @@ other Nussknacker non K8s configuration.
 
 | Parameter                     | Type                                                | Default value                     | Description                                                                              |
 |-------------------------------|-----------------------------------------------------|-----------------------------------|------------------------------------------------------------------------------------------|
-| mode                          | string                                              |                                   | Either streaming or request-response                                                     |
+| mode                          | string                                              |                                   | (Processing) mode: either streaming or request-response                                                     |
```

Comment:
we have also a concept of the _mode_ in the Helm chart (slightly different meaning). Wanted to clarify this.

Reply:
removed ()